utils/make-sqlite.py

```diff
@@ -89,7 +97,14 @@
     ruleset_id = c.lastrowid
     for target in targets:
         c.execute('''INSERT INTO targets (host, ruleset_id) VALUES(?, ?)''', (target, ruleset_id))
```

Comment:
I kept the sqlite generation around because the validator still uses it. Eventually I'd like to break that dependency, but it will be a bigger, harder change, and I wanted to get this out for the next release so Mozilla will start accepting our releases again.